sentry/Dockerfile

```diff
@@ -1,5 +1,6 @@
 ARG SENTRY_IMAGE
-FROM ${SENTRY_IMAGE}
+ARG SENTRY_PYTHON3
+FROM ${SENTRY_IMAGE}${SENTRY_PYTHON3:+-py3}
```

Comment:
Huh, TIL docker has limited support for shell-style envvar expansion. https://docs.docker.com/engine/reference/builder/#environment-replacement